File: 1-easy/problems/3894-traffic-signal-color/solution.py
class Solution:
    def trafficSignal(self, timer):
        """
        :type timer: int
        :rtype: str
        """
        if timer == 0:
            return "Green"
        elif timer == 30:
            return "Orange"
        elif 30 < timer <= 90:
            return "Red"
        return "Invalid"

File: 1-easy/problems/3894-traffic-signal-color/solution.ts
function trafficSignal(timer: any): boolean | number | string | any {
  """
  :type timer: int
  :rtype: str
  """
  if (timer == 0) {
      return "Green";
  } else if (timer == 30:
      return "Orange";
  } else if (30 < timer <= 90:
      return "Red";
  return "Invalid";
}

export { trafficSignal };
